src/components/Settings/Settings.js

```diff
@@ -70,6 +70,14 @@ class Settings extends React.Component {
       });
   }
 
+  componentDidMount() {
+    if (this.paneTitleRef.current) {
+      this.paneTitleRef.current.focus();
+    }
+  }
+
+  paneTitleRef = createRef();
```

Comment:
@Javokhir12 since we already have a constructor here would you mind moving it there? It's just easier to reason about.

Reply:
@mkuklis good point, it is done.